Commands/Commands.cs

```diff
@@ -23,5 +23,14 @@ public async Task GetLatency(CommandContext ctx)
                 .Build();
             await ctx.Channel.SendMessageAsync(embed: latency);
         }
+        [Command("F")]
+        private async Task F()
```

Comment:
Making a task private will hide it and hence it won't be called.

Reply:
Oops I missed that